components/AppBar/src/MDCAppBarViewController.h

```diff
@@ -17,6 +17,7 @@
 #import "MaterialNavigationBar.h"
 
 @class MDCAppBarViewController;
+@protocol MDCAppBarViewControllerDelegate;
```

Comment:
Should the name of the protocol be more specific? Like `MDCAppBarViewControllerAccessibilityDelegate` or something?

Reply:
I'm on the fence about the granularity of delegates, but it looks like FlexibleHeader's already sliding towards granular delegates in general so I'll keep that convention here.